components/items/ItemsGrid/useNftActions.ts

```diff
@@ -40,6 +40,12 @@ export const prepareListingQuery = (
         token: tokenSearchTerm,
         currentOwner_eq: accountId.value,
         burned_eq: false,
+        OR: {
+          price_isNull: true,
+          token: tokenSearchTerm,
+          currentOwner_eq: accountId.value,
+          burned_eq: false,
+        },
```

Comment:
I'm confused about this OR condition here. Would it be possible to remove the price limit (lines 53- 54) instead?

<img width="640" alt="image" src="https://github.com/user-attachments/assets/0b5d862e-9c50-4cdc-affe-efbf3ad4d6b1">

Reply:
if there are no edge case Ok